python/graphstorm/sagemaker/sagemaker_partition.py

```diff
@@ -214,7 +214,8 @@ def run_partition(job_config: PartitionJobConfig):
     metadata_filename = job_config.metadata_filename
     skip_partitioning = job_config.skip_partitioning == 'true'
 
-    sm_env = json.loads(os.environ['SM_TRAINING_ENV'])
+    with open("/opt/ml/config/resourceconfig.json", "r", encoding="utf-8") as f:
```

Comment:
Is SageMaker API changed?

Reply:
No the launch calls remain the same from the user's perspective. The information that exists as env vars in Training exists in this file in Processing jobs.